test/helpers/kubectl.go

```diff
@@ -627,6 +627,24 @@ func (kub *Kubectl) GetCiliumEndpoint(namespace string, pod string) (*cnpv2.Endp
 	return data, nil
 }
 
+// GetCiliumHostEndpointID returns the ID of the host endpoint on a given node.
+func (kub *Kubectl) GetCiliumHostEndpointID(ciliumPod string) (int64, error) {
+	cmd := fmt.Sprintf("%s -n %s exec %s -- cilium endpoint list -o jsonpath='{[?(@.status.identity.id==%d)].id}'",
+		KubectlCmd, CiliumNamespace, ciliumPod, ReservedIdentityHost)
```

Comment:
Shouldn't we use `GetCiliumNamespace(GetCurrentIntegration())` instead of `CiliumNamespace`?